src/apps/common/i18n/en-US.json

```
@@ -648,7 +648,12 @@
 		"silence": "Silence",
 		"shoutcastURLWarning": "WARNING: If the stream selected here attempts to be played, and is unavailable, the call may fail completely. Please make sure the stream you use is available 24/7.",
 		"useExistingMedia": "Use Existing Media",
-		"uploadNewMedia": "Upload New Media"
+		"uploadNewMedia": "Upload New Media",
+		"textToSpeech": {
+			"tab": "Text-to-Speech",
+			"info": "Customize your greeting message using Text-to-Speech. Type your desired greeting below. Once your changes are saved, anyone who calls this number will be greeted with your custom message.",
```

We might want to insert a variable in place of `"number"` in this sentence, as TTS could be used for different entities (number, voicemail, conference etc.).

The value of this variable could either be set by the invocator directly (e.g. they would set the i18n string to be displayed) or, we could keep a list of entities locally. Which do you thing would make more sense?

```suggestion
			"info": "Customize your greeting message using Text-to-Speech. Type your desired greeting below. Once your changes are saved, anyone who calls this {{entity}} will be greeted with your custom message.",
```

---

hey @azefiel , I think using a i18n string will be the best way, just following the way that we already handle strings in Monster UI.